### PR TITLE
Revert wlcore changes for beaglebone green wifi

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

**Comment by willnewton** [comment]
I don't see this commit in the latest 4.4 tag:

    [will@localhost beagleboard-linux]$ git status
    HEAD detached at 4.4.55-ti-rt-r94

**Comment by floion** [comment]
Didn't check the latest tag. Maybe best to check and see if the changes done by that commit are in the file @willnewton ?

**Comment by willnewton** [comment]
Ok it looks like that commit is in the tag we use currently (4.4.21-ti-rt-r46) but not in the latest 4.4 tag (4.4.55-ti-rt-r94). The version bump in https://github.com/resin-os/resin-beaglebone/pull/79 will have the same effect.

**Comment by floion** [comment]
Great. The kernel version bump though needs to be tested for a while before it gets merged.
And it may be a good idea to also test the wl18xx does not crash with that version before updating the kernel version

**Comment by willnewton** [comment]
I have tested the wifi with the kernel version bump and it does not crash. This change is smaller than the kernel bump it will just mean it has to be reverted after release.

File: layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.4/0001-Revert-bbgw-use-led-for-wl18xx_wl_en-wl18xx_bt_en.patch [hunk 1]
```diff
@@ -0,0 +1,102 @@
+From c7bc8da72a88ca77206fd530d441898135c52301 Mon Sep 17 00:00:00 2001
+From: Florin Sarbu <florin@resin.io>
+Date: Wed, 29 Mar 2017 10:02:48 +0200
+Subject: [PATCH] Revert "bbgw: use led for wl18xx_wl_en/wl18xx_bt_en"
+
+This reverts commit 84ea21e84e825ba61ca25f481f2cb8b1db0aeec0.
+---
```
Comment (michal-mazurek) [inline layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.4/0001-Revert-bbgw-use-led-for-wl18xx_wl_en-wl18xx_bt_en.patch]: No signed-of-by and upstream-status?

**Comment by floion** [comment]
@willnewton you have tested the kernel bersion bump on the beaglebone green wireless?

File: layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.4/0001-Revert-bbgw-use-led-for-wl18xx_wl_en-wl18xx_bt_en.patch (hunk 1 above)
Comment (floion) [inline layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.4/0001-Revert-bbgw-use-led-for-wl18xx_wl_en-wl18xx_bt_en.patch]: No :)

**Comment by willnewton** [comment]
I've tested on the BeagleBone Blue which has the same wireless chipset. I don't have a Green Wireless to test with.

**Comment by floion** [comment]
Great then. Means somebody from the Lindon office could just test the kernel bump on the  green and check and if it works then this PR is not needed anymore.
So confirm that the blue also uses wl18xx kernel module for the blue's chipset @willnewton ?

**Comment by willnewton** [comment]
lsmod shows:

    wl18xx                 87359  0
    wlcore                193629  1 wl18xx

**Comment by floion** [comment]
Good. Same driver

**Comment by willnewton** [comment]
This PR has been superceded by the kernel version bump.

**Pull request closed without merging**